src/m365/planner/commands/task/task-add.ts

```diff
@@ -126,10 +139,20 @@ class PlannerTaskAddCommand extends GraphCommand {
       });
   }
 
+  private generateAppliedCategories(options: Options): AppliedCategories {
+    if (!options.appliedCategories) {
+      return {};
+    }
+
+    const categories: AppliedCategories = {};
+    chatUtil.convertParticipantStringToArray(options.appliedCategories.toLocaleLowerCase()).forEach(x => categories[x] = true);
```

Comment:
I didn't mean you to use `chatUtil` here 😉, that's a shared util helper for the chat classes. I meant that you could learn from the way in which the string is split on spaces (for powershell) and comma's (for bash etc)

Reply:
Wouldn't the implementation then be the same as the function within `chatUtil`?

Reply:
That's correct, more or less, aside from naming. Maybe we can one day move it into a more central helper class.

Reply:
If you like you can also refactor the `chatUtil.convertParticipantStringToArray` function to the `formatting` util class. rename it to `splitByCommaOrSpace` and explain the usage in a good comment.

Reply:
Is this possibly something we'd better tackle in a new issue. Because the command `task set` could also use this refactor. Then the commands `chat get` and `chat message send`, can also use the function from the formatting util class. What do you think?

Reply:
I agree, we'll be touching too many commands in this one otherwise. That's not how we roll 😁.
In that case however: do remove the chatUtil class from here and just implement the function locally.

Reply:
Perfect, will do. Should I create an issue for the refactor?